lix/source-code/client/src/helpers.ts

```diff
@@ -18,7 +18,8 @@ export const withLazyFetching = ({
 }: Args): NodeishFilesystem => {
 	return new Proxy(nodeishFs, {
 		get(getTarget: typeof nodeishFs, prop, receiver) {
-			if (getTarget[prop as keyof typeof nodeishFs]) {
+			// eslint-disable-next-line no-prototype-builtins
+			if (((getTarget as any).__proto__ as any).hasOwnProperty(prop)) {
```

Comment:
why is the proto access needed? add a comment above please

Reply:
@martin-lysk could you fill in reason for this please

Reply:
I drop it